src/components/UserManagement/UserSearchPanel.jsx

```diff
@@ -1,18 +1,34 @@
-import React from 'react';
+import React, { useEffect,useState } from 'react';
 import { SEARCH, SHOW, CREATE_NEW_USER, SEND_SETUP_LINK } from '../../languages/en/ui';
 import { boxStyle } from 'styles';
 /**
  * The search panel stateless component for user management grid
  */
 const UserSearchPanel = props => {
-  // console.log('UserSearchPanel props', props);
+  const [hasRole, setHasRole] = useState(false);
+  const [isOwner, setIsOwner] = useState(false);
 
+  useEffect(() => {
+    if (props.user?.role === 'Owner'||props.user?.role === 'Administrator') {
+      setIsOwner(true)
+    } else {
+      setIsOwner(false)
+      const hasValue =props.user?.permissions.frontPermissions.some(value => value === 'postUserProfile');
+      if (hasValue) {
+        setHasRole(true);
+      } else {
+        setHasRole(false);
+      }
+    }
+  },[])
```

Comment:
```suggestion
  const canCreateUsers = props.hasPermission('postUserProfile');
```
It should look something like this.